shell/renderer/web_worker_observer.cc

```diff
@@ -15,28 +18,31 @@ namespace electron {
 
 namespace {
 
-static base::LazyInstance<
-    base::ThreadLocalPointer<WebWorkerObserver>>::DestructorAtExit lazy_tls =
-    LAZY_INSTANCE_INITIALIZER;
+static base::NoDestructor<base::ThreadLocalOwnedPointer<WebWorkerObserver>>
+    lazy_tls;
 
 }  // namespace
 
 // static
 WebWorkerObserver* WebWorkerObserver::GetCurrent() {
-  WebWorkerObserver* self = lazy_tls.Pointer()->Get();
-  return self ? self : new WebWorkerObserver;
+  return lazy_tls.Pointer()->Get();
```

Comment:
```suggestion
  return lazy_tls->Get();
```

other places as well